build.gradle

```diff
@@ -19,8 +19,8 @@ allprojects {
 }
 
 ext {
-    compileSdkVersion = 22
-    buildToolsVersion = "22.0.1"
+    compileSdkVersion = 23
+    buildToolsVersion = "23.0.1"
```

Comment:
Upgrading these was necessary I see, due to a compile error if left as previous values. I would say this is OK to go ahead with for now, but will want to make sure @doneill is aware of the change.